.github/pull_request_template.md

```diff
@@ -8,23 +8,14 @@
 
 <!--Give us some reference link(s) that might help quickly review and merge your PR, for example, a file link that supports why you changed the document.-->
 
-- [ ] Reference link(s):<!--Give links here-->
-- [ ] This PR is to align with:<!--Give links here-->
+- Reference link(s):<!--Give links here-->
+- This PR is to align with:<!--Give links here-->
 - [ ] N/A (not applicable)
 
-### Which TiDB version(s) does your changes apply to? <!--Required-->
+**Important Notice:**
 
-<!--Tick the checkbox(es) below to choose the TiDB version(s) that your changes apply to.-->
+**If your changes apply to multiple TiDB documentation versions**, to trigger the bot to cherry-pick this PR to other release branches, make sure you add labels such as:
 
-- [ ] All active versions: dev, v3.0, v2.1, v3.1
-- [ ] dev (the latest development version)
-- [ ] v3.0 (TiDB 3.0 versions)
-- [ ] v2.1 (TiDB 2.1 versions)
-- [ ] v3.1 (TiDB 3.1 versions)
-- [ ] v4.0 (TiDB 4.0 versions)
-- [ ] All active and inactive versions
-- [ ] N/A (not applicable)
-
-**Note:** If your changes apply to multiple TiDB versions, make sure you update the documents in the **corresponding** version folders such as "dev", "v3.0", "v2.1" and "v3.1" in this PR.
-
-- [ ] Updated one version first. Will update other versions after I get two LGTMs.
\ No newline at end of file
+- **needs-cherry-pick-3.0**
+- **needs-cherry-pick-3.1**
+- **needs-cherry-pick-2.1**
```

Comment:
Do we need to keep this line for `release-2.1`?

Reply:
No, I'll update it.